config/areas/site.php

```diff
@@ -13,26 +13,26 @@
         'routes' => [
             [
                 'pattern' => 'pages/(:any)',
-                'action'  => function (string $path) use ($kirby) {
-                    return Panel::page($path)->panel()->route();
+                'action'  => function (string $path) {
+                    return Find::page($path)->panel()->route();
                 }
             ],
             [
                 'pattern' => 'pages/(:any)/files/(:any)',
-                'action'  => function (string $id, string $filename) use ($kirby) {
-                    return Panel::file('pages/' . $id, $filename)->panel()->route();
+                'action'  => function (string $id, string $filename) {
+                    return Find::file('pages/' . $id, $filename)->panel()->route();
                 }
             ],
             [
                 'pattern' => 'site',
-                'action'  => function () use ($kirby) {
-                    return $kirby->site()->panel()->route();
+                'action'  => function () {
+                    return Find::site()->panel()->route();
```

Comment:
I think this is unnecessary - it communicates that we're doing something special here while the old version would just work fine. We aren't finding anything, just using the site object as is.

Reply:
Right now, it's totally useless. But I was thinking about the case of improved permissions. Maybe we want to disable read access for the site to some users or modify model access in a different way. But we could of course leave this for now until we really need it.

Reply:
But this shouldn't be solved in the method getting the site object, right? but then in the `Panel\Site::route` method or so

Reply:
It depends. Right now we already check for readable pages and files in `Find::page` and `Find::file` This is quite important because we can already pretend they are not even there and throw a proper NotFoundException. But for more detailed permissions checks we would indeed probably go a different way. I will remove this method for now. It's adding more confusion than actual benefits.